### PR TITLE
Improve export output

### PR DESCRIPTION
**Pull request by cleishm**
Generate neo4j-client and cypher-shell compatible output by default.
Use `db.awaitIndex(...)` rather than `schema await`.

### CODE REVIEW

File: src/test/java/apoc/export/cypher/ExportCypherTest.java [hunk 1]
```diff
@@ -27,41 +23,130 @@
...
+            return nodes() + schema() + relationships() + cleanup();
+        }
+    }
+
+    private static class CypherOutput implements ExpectedOutputs
+    {
+        @Override
+        public String nodes() {
+            return String.format(":BEGIN%n" +
+                    "CREATE (:`Foo`:`UNIQUE IMPORT LABEL` {`name`:\"foo\", `UNIQUE IMPORT ID`:0});%n" +
```
Comment (jexp) [inline src/test/java/apoc/export/cypher/ExportCypherTest.java]: I actually wanted to minimize the duplication of the generated cypher strings, that's why they've been defined only once.

File: src/main/java/apoc/export/util/ExportFormat.java [hunk 2]
```diff
@@ -36,7 +37,7 @@ public static final ExportFormat fromString(String format) {
                 }
             }
         }
-        return NEO4J_SHELL;
+        return CYPHER;
```
Comment (jexp) [inline src/main/java/apoc/export/util/ExportFormat.java]: If we're changing the default in a .dot release, we have to clearly communicate it, or leave it for 3.2

Comment (cleishm) [inline src/main/java/apoc/export/util/ExportFormat.java]: Oh, I thought this format stuff was unreleased. Given it is, then you're right - leave that change for your 3.2 release.

File: src/test/java/apoc/export/cypher/ExportCypherTest.java (hunk 1 above)
Comment (cleishm) [inline src/test/java/apoc/export/cypher/ExportCypherTest.java]: Yeah - but I don't like programmatically generating assertion text in test code. It's hard to read what the assertion should be, and you very quickly end up with test code that has the same behavior (and bugs) as the code you're intending to test.

**Comment by inserpio** [comment]
@jexp push with #380. Please note that `db.awaitIndex` is available from 3.1, so we need to remove it when merging into 3.0

**Pull request closed without merging**